src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1223,6 +1223,13 @@ int xcldev::device::runTestCase(const std::string& py,
         xclbinPath += xclbin;
 
         if (stat(xrtTestCasePath.c_str(), &st) != 0 || stat(xclbinPath.c_str(), &st) != 0) {
+            // flat shell support
+            std::string logic_uuid, errmsg;
+            pcidev::get_dev(m_idx)->sysfs_get( "", "logic_uuids", errmsg, logic_uuid);
+            if (!logic_uuid.empty() && xclbin.compare("verify.xclbin") == 0) {
```

Comment:
Add comment:
`// Only skip the test if it nonDFX platform and the accelerator doesn't exist.  All other conditions should generate an error.`